src/azure-cli/azure/cli/command_modules/network/_help.py

```diff
@@ -3785,6 +3785,88 @@
     text: az network private-link-service update -g MyResourceGroup -n MyPLSName --visibility SubId1 SubId2 --auto-approval SubId1 SubId2
 """
 
+helps['network private-endpoint-connection'] = """
+type: group
+short-summary: Manage private endpoint connections.
+"""
+
+helps['network private-endpoint-connection approve'] = """
+type: command
+short-summary: Approve a private endpoint connection.
+examples:
+  - name: Approve a private endpoint connection for a storage account.
+    text: az network private-endpoint-connection approve -g MyResourceGroup -n MyPrivateEndpoint --service-name MySA --type Microsoft.Storage/storageAccounts --description "Approved"
+  - name: Approve a private endpoint connection for a keyvault.
+    text: az network private-endpoint-connection approve -g MyResourceGroup -n MyPrivateEndpoint --service-name MyKV --type Microsoft.Keyvault/vaults --description "Approved"
```

Comment:
`service-name` is confusing, would it more clear `resource-name`?

Reply:
make senses. Will change in another PR within this release